_apidocs/get-opportunities-public-api.md

```diff
@@ -462,6 +469,9 @@ Note: Response for one record is provided as an example <br>
             "typeOfSetAside": null,
             "responseDeadLine": null,
             "naicsCode": "236220",
+			"naicsCodes":[
```

Comment:
There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?